bucket/curl.json

```diff
@@ -24,11 +24,11 @@
         "architecture": {
             "64bit": {
                 "url": "https://curl.haxx.se/windows/dl-$version/curl-$version-win64-mingw.tar.xz",
-                "extract_dir": "curl-$version-win64-mingw"
+                "extract_dir": "curl-$matchHead-win64-mingw"
             },
             "32bit": {
                 "url": "https://curl.haxx.se/windows/dl-$version/curl-$version-win32-mingw.tar.xz",
-                "extract_dir": "curl-$version-win32-mingw"
+                "extract_dir": "curl-matchHead-win32-mingw"
```

Comment:
@rasa Should this `matchHead` not have a `$` before it like the 64-bit one?

Reply:
Good catch! Fixed in https://github.com/ScoopInstaller/Main/commit/fb31dc1c9614086bc64cb4b98a68177ba6729649